Packs/ContentManagement/Scripts/CommitFiles/CommitFiles.py

```diff
@@ -133,6 +135,21 @@ def does_file_exist(branch_name: str, content_file: ContentFile) -> bool:
     return False
 
 
+def does_file_exist_azure_devops(branch_name: str, content_file: ContentFile) -> bool:
+    full_path = Path(content_file.path_to_file, content_file.file_name)
+
+    if str(full_path) in files_path:
+        return True
+    # try to get the file from branch
+    response = execute_command('azure-devops-file-list', args={'branch_name': branch_name.split('/')[-1], 'recursion_level': 'Full'})
+    files_set = {file.get("path", "") for file in response.get("value", [])}
+    for file in files_set:
+        if full_path.name in file:
```

Comment:
what if there are two files with the same name, in different paths?

Reply:
fixed